src/main/java/org/jenkinsci/plugins/DependencyTrack/ResultAction.java

```diff
@@ -15,28 +15,34 @@
  */
 package org.jenkinsci.plugins.DependencyTrack;
 
+import hudson.model.Action;
 import hudson.model.Run;
 import jenkins.model.RunAction2;
+import jenkins.tasks.SimpleBuildStep;
 import net.sf.json.JSONObject;
 import net.sf.json.JsonConfig;
 import org.jenkinsci.plugins.DependencyTrack.model.Finding;
 import org.jenkinsci.plugins.DependencyTrack.model.SeverityDistribution;
 import org.jenkinsci.plugins.DependencyTrack.transformer.FindingsTransformer;
 import org.kohsuke.stapler.bind.JavaScriptMethod;
-import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.List;
 
-public class ResultAction implements RunAction2, Serializable {
+public class ResultAction implements RunAction2, SimpleBuildStep.LastBuildAction {
```

Comment:
LastBuildAction lets jenkins lookup the ProjectActions of a BuildAction.  Also, removed the Serializable stuff since I a) couldn't figure why it was needed, and b) it made things complicated because JobAction is not Serializable.